pkg/template/controller/templateinstance_controller_test.go

```diff
@@ -24,6 +24,10 @@ import (
 	templatev1 "github.com/openshift/api/template/v1"
 )
 
+func init() {
+	batchv1.AddToScheme(legacyscheme.Scheme)
```

Comment:
This needs to be fixed (can be followup), this might risk infecting other tests. We should have scheme just for the test, as discussed some time ago.